collections/_release_4_5/entry-platforms-web-webassembly-simd-support.md

```diff
@@ -0,0 +1,18 @@
+---
+type: entry
+section: platforms
+subsection: web
+rank: 0
+importance: 2
+anchor: webassembly-simd-support
+title: WebAssembly SIMD support
+blockquote: Free performance boost
+text: |
+  For about two years now, all major browsers have supported WASM (WebAssembly) SIMD. SIMD stands for “Single instruction, multiple data” and is a technology that permits CPUs to do some parallel computation, often speeding up the whole program.
+
+  Starting with 4.5, you can expect your Web games to run a little bit more smoothly, without having to do anything. Especially when things get chaotic (for your CPU). It isn’t a silver bullet for poorly optimized games, but it will help nonetheless. Also, note that it cannot do anything for GPU rendering bottlenecks.
```

Comment:
```suggestion
  Starting with 4.5, you can expect your Web games to run a little bit more smoothly, without having to do anything – especially when things get chaotic (for your CPU). It isn’t a silver bullet for poorly optimized games, but it will help nonetheless. Also, note that it cannot do anything for GPU rendering bottlenecks.
```
I feel like the last two sentences might diminish the excitement/impact of the feature being described. They sound like valid points, but I could see someone reading this section and their lasting impression being "oh, but there are caveats..." rather than "wow, generally speaking my web games will run better!"